.gitignore

```diff
@@ -9,3 +9,5 @@
 # Temporary output of build tools
 bazel-*
 *.out
+
+tmp
```

Comment:
Not mine. This tmp dir gets created during the code generation. Because it was taking so long/freezing on my machine and I had to kill that process many times, I noticed that depending on how far in the process the dir wouldn't have been cleaned up. 

I could have added that to my global gitignore but figured it will be useful in the case when interrupting the codegen before it runs to the end.